dist/config/doctrine.yaml

```diff
@@ -0,0 +1,19 @@
+parameters:
+    env(MYSQL_SERVER_VERSION): 'mariadb-10.5.13'
```

Comment:
Seems a bit strange to set this exact version as the default... do we really need this?

Reply:
You are right. Setting the version as default in the bundle might be not good, as we have to care about updating.

I tried to run my Docker Compose setup without telling the server version and it worked. So maybe for our approach we can just remove the default value.

The docs say the server_version is needed when not database was initialized, because then the version cannot be guessed. But my setup does create an initial database at startup (unintentionally), so i think we are save for the moment.